app/classifier/tasks/drawing/markings-renderer.cjsx

```diff
@@ -18,9 +18,8 @@ module.exports = React.createClass
     # console.log 'Old marks were', @state.oldSetOfMarks
     newSetOfMarks = []
     # Automatically select new marks.
-    annotations = nextProps.classification?.annotations ? []
-    annotation = annotations[annotations.length - 1]
-    if annotation?
+    annotation = nextProps.annotation
```

Comment:
I had a search through the code and the annotation is passed down here, so that's fine. https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/classifier/classifier.jsx#L216

`MarkingRenderer` shouldn't have been using `classification.annotations` to get something that was already in the props.